test/unit/shady-unscoped-style.html

```diff
@@ -99,26 +99,12 @@
   var el1 = document.querySelector('my-element');
   var el2 = document.querySelector('my-element2');
 
-  test('unscoped styles apply', function() {
-    assertComputed(el1.$.fromStyle, '1px');
-    assertComputed(el1.$.fromImport, '2px');
-    assertComputed(el2.$.fromStyle, '1px');
-    assertComputed(el2.$.fromImport, '2px');
-  });
```

Comment:
Why are all these tests removed?  The `fromStyle` ones seem to just be style module-related...

Reply:
good point.

Reply:
The second set are deleted because there is nothing to dedupe since we don't have the css html import

Reply:
relevant tests added back in